tests/modules/gatk4/mergebamalignment/main.nf

```diff
@@ -14,3 +14,14 @@ workflow test_gatk4_mergebamalignment {
 
     GATK4_MERGEBAMALIGNMENT ( input, fasta, dict )
 }
+
+workflow test_gatk4_mergebamalignment_stubs {
+    input    = [ [ id:'test' ], // meta map
+                 file(params.test_data['sarscov2']['illumina']['test_single_end_bam'], checkIfExists: true),
+                 file(params.test_data['sarscov2']['illumina']['test_unaligned_bam'], checkIfExists: true)
+               ]
+    fasta    = file(params.test_data['sarscov2']['genome']['genome_fasta'], checkIfExists: true)
+    dict     = file(params.test_data['sarscov2']['genome']['genome_dict'], checkIfExists: true)
+
+    GATK4_MERGEBAMALIGNMENT ( input, fasta, dict )
```

Comment:
If we tests stubs, do we need an actual real file?
Could this be working?
```suggestion
    input    = [ [ id:'test' ], // meta map
                 "test_foo.bam",
                 "test_bar.bam"
               ]
    fasta    = "genome.fasta"
    dict     = "genome.fasta.dict"

    GATK4_MERGEBAMALIGNMENT ( input, fasta, dict )
```

Reply:
Good point! I have updated it on all the modules changed.

Reply:
If this works, do you think this should be the standard for testing stubs? ;) The advantage I see for not using actual files is that they wouldn't have to be staged in the work folder, so we get a leaner work environment.

Reply:
Definitively, never really thought of that before actually seeing that here

Reply:
It seems to work ^^